mandatory/2-function-creation.js

```diff
@@ -17,93 +17,66 @@ Tip: use logical operators
-- the numbers must be rounded to 2 decimal places
+- takes a number as input
+- return a string formatted as percentages (e.g. 10 => "10%")
+- the number must be rounded to 2 decimal places
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(arr) {}
+function formatPercentage(num) {}
 
 /* ======= TESTS - DO NOT MODIFY ===== */
 
-const util = require("util");
-
-function test(test_name, actual, expected) {
-  let status;
-
-  let isEqual;
-  if (Array.isArray(expected)) {
-    isEqual = arraysEqual(actual, expected);
-  } else {
-    isEqual = actual === expected;
-  }
-
-  if (isEqual) {
-    status = "PASSED";
-  } else {
-    status = `FAILED: expected: ${util.inspect(
-      expected
-    )} but your function returned: ${util.inspect(actual)}`;
-  }
-
-  console.log(`${test_name}: ${status}`);
-}
-
-function arraysEqual(a, b) {
-  if (a === b) return true;
-  if (a == null || b == null) return false;
-  if (a.length != b.length) return false;
-
-  for (let i = 0; i < a.length; ++i) {
-    if (a[i] !== b[i]) return false;
-  }
-
-  return true;
-}
-
-test(
-  "tidyUpString function works - case 1",
-  tidyUpString(["/Daniel ", "irina ", " Gordon", "ashleigh "]),
-  ["daniel", "irina", "gordon", "ashleigh"]
-);
-test(
-  "tidyUpString function works - case 2",
-  tidyUpString([" /Sanyia ", " Michael ", "AnTHonY ", "   Tim   "]),
-  ["sanyia", "michael", "anthony", "tim"]
-);
-
-test("validate function works - case 1", validate(10), true);
-test("validate function works - case 2", validate(18), true);
-test("validate function works - case 3", validate(17), false);
-test("validate function works - case 4", validate("Ten"), false);
-test("validate function works - case 5", validate(108), false);
-
-test("remove function works - case 1", remove([10, 293, 292, 176, 29], 3), [
-  10,
-  293,
-  292,
-  29,
-]);
-test(
-  "remove function works - case 2",
-  remove(["a", "b", "c", "d", "e", "f", "g"], 6),
-  ["a", "b", "c", "d", "e", "f"]
-);
-
-test(
-  "formatPercentage function works - case 1",
-  formatPercentage([23, 18.103, 187.2, 0.372]),
-  ["23%", "18.1%", "100%", "0.37%"]
-);
+const {expect, test} = require("@jest/globals");
+
+test.each([
+  ["/Daniel", "daniel"],
+  [" Sanyia/", "sanyia"],
+  ["AnTHonY", "anthony"],
+  ["irina", "irina"],
+  [" Gordon", "gordon"],
+  ["ashleigh  ", "ashleigh"],
+  ["  Alastair  ", "alastair"],
+  [" anne marie  ", "anne marie"],
+])("tidyUpString function works for %s", (input, expected) => {
+    expect(tidyUpString(input)).toEqual(expected);
+});
+
+test("validate function accepts valid even number", () => {
+    expect(validate(10)).toEqual(true);
+});
+
+test("validate function accepts other valid even number", () => {
+    expect(validate(18)).toEqual(true);
+});
+
+test("validate function accepts exactly 100", () => {
+    expect(validate(100)).toEqual(true);
+});
+
+test("validate function works rejects odd number", () => {
```

Comment:
Typo in the title here

```suggestion
test("validate function rejects odd number", () => {
```